src/asset-management/EulerV2Manager.sol

```diff
@@ -305,17 +263,61 @@ contract AaveManager is IAssetManager, Owned(msg.sender), ReentrancyGuard {
                                 ADDITIONAL REWARDS
     //////////////////////////////////////////////////////////////////////////*/
 
-    function claimRewardForMarket(address aMarket, address aReward)
+    function claimRewards(
+        IDistributor aDistributor,
+        address[] calldata aUsers,
+        address[] calldata aTokens,
+        uint256[] calldata aAmounts,
+        bytes32[][] calldata aProofs
+    ) external onlyGuardianOrOwner {
```

Comment:
This is the interface for euler, right? If we need to change we can just do windown mode and migrate?

Reply:
this is the interface for merkl, a rewards distribution protocol which euler and its partners have chosen to distribute additional incentives (on top of the base yield). think of it as minichef as a service, fuss free. 

Yes exactly when needed, we can always wind down and migrate. If there rewards not claimable by this function, we can always use the `rawCall` function

Reply:
Perfect sounds good